snap/snapcraft.yaml

```diff
@@ -12,11 +12,16 @@ description: |
 
   Commands can be classified as follows:
 
-    Basic: help, version
-    Action: cut
+    Basic (general operations):
```

Comment:
Why do we have this information in the snap description at all?  This is not conventional for snaps or packages in general.  I suggest doing the usual and stopping at line 10 above.

Reply:
Thanks, this seems better. I have removed the usage info from the description in [4ca2eeb](https://github.com/canonical/chisel/pull/165/commits/4ca2eeb8e978d88eeda2b5a1af3368c2d88c3ae7).